src/main/java/core/basesyntax/exception/IllegalDateParametersException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class IllegalDateParametersException extends Exception {
+    public IllegalDateParametersException(String exceptionMessage) {
+        super("Wrong parameters!");
```

Comment:
still constructor accepting exceptionMessage from here `throw new IllegalDateParametersException("Wrong parameters!");` and then what?

Reply:
done :)